bootstrap-app/src/cmr/bootstrap/system.clj

```diff
@@ -35,6 +35,7 @@
    [cmr.metadata-db.config :as mdb-config]
    [cmr.metadata-db.system :as mdb-system]
    [cmr.metadata-db.services.util :as mdb-util]
+   [cmr.search.services.query-execution.has-granules-results-feature :as has-granules-results-feature]
```

Comment:
you could say has-gran-rslt-feat, it's a bit long, but I don't mind either way. I love how people use longer names these days, but still remember fondly when you could shorting things up